clang/lib/CodeGen/CGHLSLRuntime.cpp

```diff
@@ -66,17 +66,17 @@ void addDxilValVersion(StringRef ValVersionStr, llvm::Module &M) {
   DXILValMD->addOperand(Val);
 }
 
-void addRootSignature(ArrayRef<llvm::hlsl::rootsig::RootElement> Elements,
+void addRootSignature(llvm::dxbc::RootSignatureVersion RootSigVer,
+                      ArrayRef<llvm::hlsl::rootsig::RootElement> Elements,
                       llvm::Function *Fn, llvm::Module &M) {
   auto &Ctx = M.getContext();
+  IRBuilder<> Builder(Ctx);
```

Comment:
`IRB` is also a common name for IRBuilders. Might make the code slightly clearer to have `IRB` and `RSBuilder` so you never need to wonder which builder `Builder` is when reading this.